scripts/check-semconv-version.sh

```diff
@@ -10,12 +10,12 @@ done < <(find . -type f -name "*.go" -exec grep -o -H "go.opentelemetry.io/otel/
 semconv_versions=($(printf "%s\n" "${semconv_map[@]}" | sort -u))
 
 if [ ${#semconv_versions[@]} -gt 1 ]; then
-    echo "Error: semconv version mismatch detected"
+    echo "Error: semconv version mismatch detected. Run ./scripts/update-semconv-version.sh to update semconv to latest version."
```

Comment:
I would recommend printing "Run <script>" line at the end of the output, after the listing of files.